pkg/keyset/key_set.go

```diff
@@ -3,8 +3,8 @@ package keyset
 import (
 	"context"
 	"crypto/rand"
+	wocclient "github.com/bitcoin-sv/arc/internal/woc_client"
```

Comment:
No alias needed as the rule has been excluded

Reply:
Removed